plugins/slicer/MONAILabelReviewer/Resources/UI/DialogInfo.py

```diff
@@ -0,0 +1,49 @@
+# Form implementation generated from reading ui file 'DialogInfo.ui'
+#
+# Created by: PyQt5 UI code generator 5.15.6
+#
+# WARNING: Any manual changes made to this file will be lost when pyuic5 is
+# run again.  Do not edit this file unless you know what you are doing.
+
+
+from PyQt5 import QtCore, QtWidgets
```

Comment:
What is this file for? Slicer does not use PyQt5. It uses [PythonQt](https://mevislab.github.io/pythonqt/)

Reply:
Good catch. PythonQt is already bundled with Slicer, so bringing in PyQt is unnecessary. Also, PyQt is GPL code that would taint the entire MONAILabel code base with its copyleft license.

Reply:
Done. The entire file was removed.
Thanks a lot for that catch.
That file was generated during development just for understanding purpose. Therefore, it is not necessary.